internal/cli/cmd/cluster/cmd.go

```diff
@@ -223,17 +223,17 @@ func newDestroyCmd() *cobra.Command {
 				result, err := tui.Ask(ctx, "Do you want to remove this cluster?",
 					fmt.Sprintf(`This is a destructive action.
 
-	Type %q for it to be removed.`, cluster.ClusterId), "")
+	Type %q for it to be removed.`, cluster.Cluster.ClusterId), "")
 				if err != nil {
 					return err
 				}
 
-				if result != cluster.ClusterId {
+				if result != cluster.Cluster.ClusterId {
 					return context.Canceled
 				}
 			}
 
-			if err := api.DestroyCluster(ctx, api.Endpoint, cluster.ClusterId); err != nil {
+			if err := api.DestroyCluster(ctx, api.Endpoint, cluster.Cluster.ClusterId); err != nil {
```

Comment:
Hmm, destroy only needs the ID, but now we call `List` and then `Get` during `selectClusters`.
Maybe it would make sense if `selectClusters` does not call `Get` at all. Instead, we can call `List` once and either check that all provided cluster IDs are valid or spawn a selection dialog.
Then the commands can call `Get` if they actually need it.